cmd/boltzd/boltzd_test.go

```diff
@@ -1680,3 +1827,200 @@ func TestChangePassword(t *testing.T) {
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			checkClnVersion(tt.args.info)
+		})
+	}
+}
+
+func Test_checkLndVersion(t *testing.T) {
+	type args struct {
+		info *lightning.LightningInfo
+	}
+	tests := []struct {
+		name string
+		args args
+	}{
+		// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			checkLndVersion(tt.args.info)
+		})
+	}
+}
+
+func Test_connectLightning(t *testing.T) {
+	type args struct {
+		lightning lightning.LightningNode
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *lightning.LightningInfo
+		wantErr assert.ErrorAssertionFunc
+	}{
+		// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := connectLightning(tt.args.lightning)
+			if !tt.wantErr(t, err, fmt.Sprintf("connectLightning(%v)", tt.args.lightning)) {
+				return
+			}
+			assert.Equalf(t, tt.want, got, "connectLightning(%v)", tt.args.lightning)
+		})
+	}
+}
+
+func Test_initBoltz(t *testing.T) {
+	type args struct {
+		cfg     *config.Config
+		network *boltz.Network
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *boltz.Api
+		wantErr assert.ErrorAssertionFunc
+	}{
+		// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := initBoltz(tt.args.cfg, tt.args.network)
+			if !tt.wantErr(t, err, fmt.Sprintf("initBoltz(%v, %v)", tt.args.cfg, tt.args.network)) {
+				return
+			}
+			assert.Equalf(t, tt.want, got, "initBoltz(%v, %v)", tt.args.cfg, tt.args.network)
+		})
+	}
+}
+
+func Test_initLightning(t *testing.T) {
+	type args struct {
+		cfg *config.Config
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    lightning.LightningNode
+		wantErr assert.ErrorAssertionFunc
+	}{
+		// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := initLightning(tt.args.cfg)
+			if !tt.wantErr(t, err, fmt.Sprintf("initLightning(%v)", tt.args.cfg)) {
+				return
+			}
+			assert.Equalf(t, tt.want, got, "initLightning(%v)", tt.args.cfg)
+		})
+	}
+}
+
+func Test_initOnchain(t *testing.T) {
+	type args struct {
+		cfg     *config.Config
+		network *boltz.Network
+	}
+	tests := []struct {
+		name    string
+		args    args
+		want    *onchain.Onchain
+		wantErr assert.ErrorAssertionFunc
+	}{
+		// TODO: Add test cases.
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			got, err := initOnchain(tt.args.cfg, tt.args.network)
+			if !tt.wantErr(t, err, fmt.Sprintf("initOnchain(%v, %v)", tt.args.cfg, tt.args.network)) {
+				return
+			}
+			assert.Equalf(t, tt.want, got, "initOnchain(%v, %v)", tt.args.cfg, tt.args.network)
+		})
+	}
+}
+
+func Test_waitForLightningSynced(t *testing.T) {
+	type args struct {
+		lightning lightning.LightningNode
+	}
+	tests := []struct {
+		name string
+		args args
+		want *lightning.LightningInfo
+	}{
+		// TODO: Add test cases.
```

Comment:
There are bunch of these comments in this file

Reply:
accidently had goland generate those